keps/sig-storage/1710-selinux-relabeling/kep.yaml

```diff
@@ -19,16 +19,21 @@ approvers:
 see-also:
   - /keps/sig-storage/695-skip-permission-change/README.md
 stage: alpha
-latest-milestone: "v1.24"
+latest-milestone: "v1.27"
 milestone:
   alpha: "v1.24"
-  beta: "v1.25"
-  stable: "v1.27"
+  beta: "v1.27"
```

Comment:
Also update the PRR reviewer file and look for a PRR reviewer

Reply:
added